packages/one-app-bundler/bin/bundle-one-app.js

```diff
@@ -17,13 +17,20 @@ const { promisify } = require('util');
 const webpack = promisify(require('webpack'));
 const chalk = require('chalk');
 
+const getConfigOptions = require('../utils/getConfigOptions');
 const config = require('../webpack/app/webpack.client');
 const getWebpackCallback = require('./webpackCallback');
 const postProcessOneAppBundle = require('./postProcessOneAppBundle');
 
-Promise.all([
+const configOptions = getConfigOptions();
+const disableLegacy = !configOptions.disableLegacy && process.env.NODE_ENV === 'development';
+
+const configuredPromise = disableLegacy ? Promise.all([
   webpack(config('modern')).then((stats) => getWebpackCallback('browser', false)(undefined, stats)),
   webpack(config('legacy')).then((stats) => getWebpackCallback('legacyBrowser', false)(undefined, stats)),
-]).then(postProcessOneAppBundle).catch((err) => {
+]) : Promise.resolve(
+  webpack(config('modern')).then((stats) => getWebpackCallback('browser', false)(undefined, stats)));
+
+configuredPromise.then(postProcessOneAppBundle).catch((err) => {
```

Comment:
all these can be reverted. we only want to disable for module builds. this is for building one-app and only affects one app development. getConfigOptions is for module builds